js/app/Visualization/UI/style.less

```diff
@@ -626,7 +626,7 @@ hyperwall is: 3 1280px screens wide and 3 720pxs screens high.
   max-width: 551px;
   height: 100%;
   padding-bottom: 135pt;
-  z-index: 10000;
+  z-index: 1;
```

Comment:
I have tested locally. Yes it does sound like that, doesn't it? But it comes from here: https://github.com/GlobalFishingWatch/pelagos-client/commit/2e1796ecd22257556b4e6378837be3ef879fddbd#diff-52e61d11d683ca62097a13f72cf34713R153 which is where the new sidebar got added, so it never got added later on in a bugfix commit or anything...